src/main/scala/io/sdkman/repos/VersionsRepo.scala

```diff
@@ -15,7 +15,10 @@ trait VersionsRepo {
 
   def findAllVersionsByCandidatePlatform(candidate: String, platform: String): Future[Seq[Version]] =
     versionsCollection
-      .find(and(equal("candidate", candidate), or(equal("platform", platform), equal("platform", "UNIVERSAL"))))
+      .find(and(
+        equal("candidate", candidate),
```

Comment:
A tiny thing, but can we have `and` on the subsequent line. All the lines below start with a boolean operator.

Reply:
No problem!